storio-common/src/test/java/com/pushtorefresh/storio/operations/internal/OnSubscribeExecuteAsBlockingCompletableTest.java

```diff
@@ -0,0 +1,64 @@
+package com.pushtorefresh.storio.operations.internal;
+
+import com.pushtorefresh.storio.StorIOException;
+import com.pushtorefresh.storio.operations.PreparedWriteOperation;
+
+import org.junit.Test;
+
+import rx.Completable;
+import rx.observers.TestSubscriber;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+public class OnSubscribeExecuteAsBlockingCompletableTest {
+
+    @SuppressWarnings("ResourceType")
+    @Test
+    public void shouldExecuteAsBlockingAfterSubscription() {
+        final PreparedWriteOperation preparedOperation = mock(PreparedWriteOperation.class);
+
+        TestSubscriber testSubscriber = new TestSubscriber();
+
+        verifyZeroInteractions(preparedOperation);
```

Comment:
This makes no sense because you don't pass it anywhere so it won't be invoked, can you move this line after `OnSubscribeExecuteAsBlockingCompletable.newInstance(preparedOperation)` but before `.subscribe()`?